src/ads/ima/manager.js

```diff
@@ -99,7 +101,7 @@ Scoped.define("module:Ads.IMA.AdsManager", [
                 }
 
                 // if size query param is not on the ad tag url, define them
-                if (!requestAdsOptions?.adTagParams?.sz.length > 0) {
+                if (!requestAdsOptions?.adTagParams?.sz) {
```

Comment:
The previous change assumes that sz will always have a length, which could cause errors. The new change will account for falsey values and a length of 0 as well.